TransformationSystem/Agent/TaskManagerAgentBase.py

```diff
@@ -195,6 +199,8 @@ def execute(self):
         self._addOperationForTransformations(operationsOnTransformationDict, 'submitTasks', transformations,
                                              owner=owner, ownerGroup=ownerGroup, ownerDN=ownerDN)
 
+    self.maxParametricJobs = Operations().getValue("JobScheduling/MaxParametricJobs", self.maxParametricJobs)
```

Comment:
Does it run with a shifter proxy ? otherwise I suppose the vo parameter should be given to Operations

Reply:
It CAN run with a shifter proxy, but not necessarily.

Reply:
Still this does not answer the remark. The Operations object must be created with the vo argument corresponding to the group of the owner of the transformation.

Reply:
Sure! I answered quickly in the weekend, I will fix it now.